metadata_tools.py

```diff
@@ -39,173 +29,126 @@ def is_file_larger_than(self, filepath: str, size_in_mb) -> bool:
         # Convert the size to megabytes
         size_in_mb_actual = size_in_bytes / (1024 * 1024)
 
+        print(size_in_mb_actual)
+
         # Compare the actual size with the specified size
         return size_in_mb_actual > size_in_mb
 
```

Comment:
if your'e leaving a block of commented code in place, put a header about why we're keepoing it. Or trash it.